src/SAWScript/Crucible/Common/Override.hs

```diff
@@ -180,10 +180,10 @@ ppOverrideFailureReason ::
   ) => OverrideFailureReason ext -> PP.Doc
 ppOverrideFailureReason rsn = case rsn of
   AmbiguousPointsTos pts ->
-    PP.text "ambiguous collection of points-to assertions" PP.<$$>
+    PP.text "LHS of points-to assertion(s) not reachable via points-tos from inputs/outputs:" PP.<$$>
```

Comment:
"not reachable" -> "not uniquely determined"?

Reply:
Is there a situation where a variable could be *determined*, but not *uniquely* so?